src/renderer/components/+workloads-deployments/deployment-scale-dialog.test.tsx

```diff
@@ -128,33 +128,45 @@ describe("<DeploymentScaleDialog />", () => {
     const initReplicas = 1;
 
     deploymentApi.getReplicas = jest.fn().mockImplementationOnce(async () => initReplicas);
```

Comment:
I wonder if we can use getBy* queries in at least most of the times in this test

Reply:
@panuhorsmalahti Thank you for your comment 🙂 I checked and it seems `getByTestId()` works fine. I will replace `findByTestId` if it is a more convenient method. Please let me know

Reply:
I'd use getBy* over findBy* if the getBy* queries work, since to me findBy* implies that the "waiting for up to 1000ms" is required for the test. Both are valid though.